app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt

```diff
@@ -55,6 +62,9 @@ class HomeActivityTest {
   fun tearDown() {
     IdlingRegistry.getInstance().unregister(MainThreadExecutor.countingResource)
   }
+  @get:Rule
+  var activityRule: ActivityTestRule<HomeActivity>
```

Comment:
I don't believe we need this since this class is now explicitly launching the HomeActivity. With this rule in place, we'll be launching it twice.

Reply:
okay